client/actions/lists.js

```diff
@@ -1,12 +1,12 @@
 export const SUBMIT_LIST = 'SUBMIT_LIST';
-export const submitList = (items, budget, deliveryAddress) => (dispatch, getState) => {
+export const submitList = (items, budget, deliveryAddress, preferredStorePlaceIds) => (dispatch, getState) => {
```

Comment:
haha and we have a collision right out of the gate, oh well

Reply:
Probably shoulda made the parameter an object to start with 🤔